tests/contrib/hooks/bigquery_hook.py

```diff
@@ -108,9 +108,30 @@ class TestBigQueryHookSourceFormat(unittest.TestCase):
     def test_invalid_source_format(self):
         with self.assertRaises(Exception) as context:
             hook.BigQueryBaseCursor("test", "test").run_load("test.test", "test_schema.json", ["test_data.json"], source_format="json")
-        
+
         # since we passed 'json' in, and it's not valid, make sure it's present in the error string.
         self.assertIn("json", str(context.exception))
 
+class TestBigQueryBaseCursor(unittest.TestCase):
+    def test_invalid_schema_update_options(self):
+        with self.assertRaises(Exception) as context:
+            hook.BigQueryBaseCursor("test", "test").run_load(
+                "test.test",
+                "test_schema.json",
+                ["test_data.json"],
+                schema_update_options=["THIS IS NOT VALID"]
+                )
+        self.assertIn("THIS IS NOT VALID", str(context.exception))
+    def test_ivalid_schema_update_and_write_disposition(self):
```

Comment:
Ugh.  For some reason my Flake8 and Pep8 don't seem to be catching some of these formatting issues.  The typo is totally my bad though.  Will fix!